bindings/aws/s3/s3.go

```diff
@@ -178,9 +182,29 @@ func (s *AWSS3) create(ctx context.Context, req *bindings.InvokeRequest) (*bindi
 		return nil, fmt.Errorf("s3 binding error. Uploading: %w", err)
 	}
 
+	var presignURL string
+	if metadata.PresignTTL != "" {
+		d, parseErr := time.ParseDuration(metadata.PresignTTL)
+		if parseErr != nil {
+			return nil, fmt.Errorf("se binding error. Cannot parse duration %s: %s", metadata.PresignTTL, parseErr)
+		}
+
+		req, _ := s.s3Client.GetObjectRequest(&s3.GetObjectInput{
+			Bucket: aws.String(metadata.Bucket),
+			Key:    aws.String(key),
+		})
+		url, signErr := req.Presign(d)
+		if signErr != nil {
+			return nil, fmt.Errorf("s3 binding error. Failed to presign URL: %s", signErr)
```

Comment:
Nit: the other errors use `:` such as `s3 binding error:`

Reply:
There are errors using both formats. I changed the errors in the `create` method now to be consistent.